platforms/corda-ent/charts/float/templates/service.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.nodeName }}-internal
+  namespace: {{ .Values.metadata.namespace}}
+  labels:
+    run: {{ .Values.nodeName }}
+    app.kubernetes.io/name: {{ .Values.nodeName }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+spec:
+  selector:
+    app: {{ .Values.nodeName }}
+# we need healthCheckNodePort set to get rid of logs pollution
+{{- if (.Values.healthCheckNodePort) }}
+  healthCheckNodePort: {{ .Values.healthCheckNodePort }}
+{{- end }}  
+  ports:
+  - name: bridge-tunnel
+    protocol: TCP
+    port: {{ .Values.bridge.tunnelPort }}
+    targetPort: {{ .Values.bridge.tunnelPort }}
```

Comment:
Add newline

Reply:
Fixed in [9862bdb ](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/893/commits/9862bdbebd85ac9508aaec96a1807981550cf571)